tests/test_read_configs_include.py

```diff
@@ -185,7 +189,7 @@ def test_delayed_eval_custom_value(self, read_config_include_file) -> None:
 
     def test_delayed_eval_quoting(self, read_config_include_file) -> None:
         """Test that the delayed evaluation functions resolve quotation marks and braces properly"""
-        extra = {"EXTRA_TEST_VAR": "\"{'123'}\"", "DELAYED_EVAL": "$DELAYED_EVAL EXTRA_TEST_VAR"}
+        extra = {"EXTRA_TEST_VAR": "\"{'123'}\"", "DELAYED_EVAL": "\"$DELAYED_EVAL EXTRA_TEST_VAR\""}
```

Comment:
Just curious, what is the unexpected conditions that needs the extra `\"` around `$DELAYED_EVAL EXTRA_TEST_VAR`?  Empty `$DELAYED_EVAL EXTRA_TEST_VAR`?

Reply:
With the quotes, the temporary env.local looks like:

```sh
EXTRA_TEST_VAR="{'123'}"
DELAYED_EVAL="$DELAYED_EVAL EXTRA_TEST_VAR"
```

without the quotes:

```sh
EXTRA_TEST_VAR="{'123'}"
DELAYED_EVAL=$DELAYED_EVAL EXTRA_TEST_VAR
```

So when this is sourced, the second line tries to run the `EXTRA_TEST_VAR` command (which doesn't exist).

Without `set -e`, this error does not stop the script from running and the test passes (this test checks that the value of `EXTRA_TEST_VAR` does *not* change).

With `set -e`, the test fails at that line since the command cannot be found.